modules/addingBoookToHtml.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] [REQUIRED] Kindly enable eslint. Eslint helps us ensure that we write a standard code. Please fix the linter errors manually.